components/authz-service/server/v2/policy_refresher.go

```diff
@@ -299,3 +299,17 @@ func (refresher *policyRefresher) getRuleMap(ctx context.Context) (map[string][]
 func pretty(vsn api.Version) string {
 	return fmt.Sprintf("IAM v%d.%d", int32(vsn.Major), int32(vsn.Minor))
 }
+
+type mockPolicyRefresher struct{}
+
+func NewMockPolicyRefresher() PolicyRefresher {
+	return &mockPolicyRefresher{}
+}
+
+func (refresher *mockPolicyRefresher) Refresh(context.Context) error {
```

Comment:
[nit] no need to name the receiver if it's not used
```suggestion
func (*mockPolicyRefresher) Refresh(context.Context) error {
```